### PR TITLE
[doc] [Transform-sql-Use UDAF] The demo on the official website has incorrect configuration parameters

### LINKED ISSUE #3355: [Bug] [Transform-sql-Use UDAF] The demo on the official website has incorrect configuration parameters
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

https://seatunnel.apache.org/docs/2.1.3/transform/sql

The configuration parameters of the sql module demo in transform on the official website are incorrect.

### SeaTunnel Version

seatunnel 2.1.3

### SeaTunnel Config

```conf
xx
```

### Running Command

```shell
xx
```

### Error Exception

```log
xx
```

### Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Carl-Zhou-CN**
…ncorrect configuration parameters

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
close #3355

### CODE REVIEW

**Comment by Carl-Zhou-CN** [comment]
@Hisoka-X @CalvinKirs Please help to review it,thanks

File: docs/en/transform/sql.md [hunk 1]
```diff
@@ -56,7 +56,6 @@ Use SQL plugin for data aggregation, use avg functions to perform aggregation op
 ```bash
 sql {
     sql = "select avg(age) from user_info",
-    table_name = "user_info"
```
Comment (Hisoka-X) [inline docs/en/transform/sql.md:59]: change `table_name` to `result_table_name` ?

Comment (Carl-Zhou-CN) [reply docs/en/transform/sql.md:59]: > change `table_name` to `result_table_name` ?

thank you,in common-options.mdx, there is a pair of results_ table_ name,source_ table_name unified description of name,I think this parameter should be added to all Sql Transforms or not. What do you think
http://seatunnel.incubator.apache.org/docs/2.3.0-beta/transform/common-options

Comment (Hisoka-X) [reply docs/en/transform/sql.md:59]: OK, remove it is fine.

**Review by Hisoka-X — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**